feat(metrics): Add metrics section (#8916) (#8969)
getsentry/sentry-docs: src/docs/product/metrics/metrics-set-up.mdx

---
title: "Set Up"
sidebar_order: 6
description: "Learn how to get started with metrics."
---

<Include name="feature-stage-alpha-metrics.mdx" />

Sentry's Developer Metrics help you track and visualize the data points you care about over time, making it easier to monitor your application health and identify issues.

To set up and start sending metrics, click a link for one of the supported SDKs below.

## Supported SDKs

Metrics are currently available on the following platforms:

> - [Python](/platforms/python/metrics/)
> - [PHP](/platforms/php/metrics/)
> - [.NET](/platforms/dotnet/metrics/)

Author: ```suggestion
- [Python](/platforms/python/metrics/)
- [JavaScript Browser](/platforms/javascript/metrics/)
- [Node](/platforms/node/metrics/)
- [PHP](/platforms/php/metrics/)
- [.NET](/platforms/dotnet/metrics/)
```

---
title: "Set Up"
sidebar_order: 6
description: "Learn how to get started with metrics."
---

> <Include name="feature-stage-beta-session-replay.mdx" />

Author: Oh, that was a big oversight on my part! Thanks for noticing, Vivian 🙏 I actually have a metrics alpha component that I'm using in all pages, just forgot to update it here.

```suggestion
<Include name="feature-stage-alpha-metrics.mdx" />
```

> To get instructions for how to set it up and start sending metrics, click a link for one of the supported SDKs below.

Author: ```suggestion
Sentry's Developer Metrics help you track and visualize the data points you care about over time, making it easier to monitor your application health and identify issues.

To set up and start sending metrics, click a link for one of the supported SDKs below.
```

To set up and start sending metrics, click a link for one of the supported SDKs below.

## Supported SDKs

Metrics are currently available on the following platforms:

> - [Python](/platforms/python/metrics/)
> - [PHP](/platforms/php/metrics/)
> - [.NET](/platforms/dotnet/metrics/)
> 
> // Links to be added

Author: ```suggestion
- [Python](/platforms/python/metrics/)
- [PHP](/platforms/php/metrics/)
- [.NET](/platforms/dotnet/metrics/)
```

> - [Python](/platforms/python/metrics/)
> - [JavaScript Browser](/platforms/javascript/metrics/)
> - [Node](/platforms/node/metrics/)
> - [PHP](/platforms/php/metrics/)
> - [.NET](/platforms/dotnet/metrics/)

Author: ```suggestion
- [Python](/platforms/python/metrics/)
- [PHP](/platforms/php/metrics/)
- [.NET](/platforms/dotnet/metrics/)
- [Vanilla JavaScript](/platforms/javascript/metrics/)
- [Node](/platforms/node/metrics/)
- [Next.js](/platforms/javascript/guides/nextjs/metrics/)
- [SvelteKit](/platforms/javascript/guides/sveltekit/metrics/)
- [Remix](/platforms/javascript/guides/remix/metrics/)
- [Astro](/platforms/javascript/guides/remix/astro/)
- [Electron](/platforms/javascript/guides/electron/metrics/)
```

---
title: "Set Up"
sidebar_order: 6
description: "Learn how to get started with metrics."
---

<Include name="feature-stage-alpha-metrics.mdx" />

Sentry's Developer Metrics help you track and visualize the data points you care about over time, making it easier to monitor your application health and identify issues.

To set up and start sending metrics, click a link for one of the supported SDKs below.

## Supported SDKs

Metrics are currently available on the following platforms:

> - Python
> - JavaScript
> - PHP
> - .NET
> - Rust

Author: ```suggestion
- [Python](/platforms/python/metrics/)
- [JavaScript](/platforms/javascript/metrics/)
- [PHP](/platforms/php/metrics/)
- [.NET](/platforms/php/metrics/)
- [Rust](/platforms/php/metrics/)
```

> - [Python](/platforms/python/metrics/)
> - [JavaScript](/platforms/javascript/metrics/)
> - [PHP](/platforms/php/metrics/)
> - [.NET](/platforms/php/metrics/)
> - [Rust](/platforms/php/metrics/)

Author: ```suggestion
- [Python](/platforms/python/metrics/)
- [PHP](/platforms/php/metrics/)
- [.NET](/platforms/dotnet/metrics/)
```
